src/main/resources/application.yaml

```diff
@@ -197,4 +197,7 @@ professional:
       scheduling:
         enabled: ${PROFESSIONAL_SCHEDULING_ENABLED:false}
         findUsersWithStaleOrganisations:
-          cron: ${CRON_P4_USERS_STALE_ORGS:*/30 * * * * *}
\ No newline at end of file
+          cron: ${CRON_P4_USERS_STALE_ORGS:*/30 * * * * *}
+      retryOneInterval: ${RETRY_ONE_INTERVAL:2}
+      retryTwoInterval: ${RETRY_TWO_INTERVAL:15}
+      retryThreeInterval: ${RETRY_THREE_INTERVAL:60}
```

Comment:
Could consider putting these inside the `findUsersWithStaleOrganisations` so we can independently configure the retries across this and the other process(es) that need retries it in future. Perhaps also a comment in here just to say that it's in minutes so anyone looking back at this in future doesn't need to check the implementation.

Reply:
maybe its worth labelling the key as retryOneIntervalMinutes...

Reply:
moved under `findUsersWithStaleOrganisations` & updated name to `retryOneIntervalMin` in [a5edcef](https://github.com/hmcts/am-org-role-mapping-service/pull/1788/commits/a5edcefe4b742fde986d16b64c99503a97c643f2)